Week3/homework/step2-6.js

```diff
@@ -4,18 +4,19 @@ const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 function flattenArray2d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  return arr.flat();
```

Comment:
Very smart to look for functions that do all the work 👍 But this function won't work on Internet Explorer and Edge, can you find a way to do this yourself and that works in all browsers?

Reply:
Thanks for your feedback! I will try to do it in another way! :) 👍